### PR TITLE
Update celery libs (bug 1136820)

### PR DESCRIPTION
**Pull request by robhudson**
I think this is ready. I tested the submission flow locally and all the tasks worked well.

### CODE REVIEW

File: settings_test.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def _polite_tmpdir():
 # is just too annoying for tests, so disable it.
 CACHE_COUNT_TIMEOUT = -1
 CELERY_ROUTES = {}
-CELERY_ALWAYS_EAGER = True
+CELERY_ALWAYS_EAGER = False
```
Comment (diox) [inline settings_test.py]: really ?

**Comment by diox** [comment]
Travis somehow didn't run on this one ?

File: settings_test.py (hunk 1 above)
Comment (robhudson) [inline settings_test.py]: I was trying to test celery locally and must have committed this on accident. I'll remove.

**Comment by robhudson** [comment]
Travis was having issues on Friday. I did a force push to try to trigger it just now.

File: settings_test.py (hunk 1 above)
Comment (diox) [inline settings_test.py]: I half hoped that there was some magic to automatically do this in tests with the latest version :)

**Comment by jasonthomas** [comment]
lgtm. I've filed https://bugzilla.mozilla.org/show_bug.cgi?id=1164050 to upgrade rabbitmq server.

**Comment by robhudson** [comment]
Merged: https://github.com/mozilla/zamboni/commit/57d310c

**Pull request closed without merging**